libs/ui-toolkit/src/components/lozenge/lozenge.tsx

```diff
@@ -0,0 +1,38 @@
+import { ReactNode } from 'react';
+import classNames from 'classnames';
+
+interface LozengeProps {
+  children: ReactNode;
+  variant?: 'success' | 'warning' | 'danger' | 'highlight';
+  className?: string;
+  details?: string;
+}
+
+const getWrapperClasses = (className: LozengeProps['className']) => {
+  return classNames('inline-flex items-center gap-4', className);
+};
+
+const getLozengeClasses = (variant: LozengeProps['variant']) => {
```

Comment:
Is this duplicated somewhere else in the ui toolkit? Can we have a helper function that takes intent and spits out background/text color classes if so

Reply:
It's not yet as far as I've seen. I'll keep an eye out and make one if we get a second instance of this though.